### PR TITLE
Added send_catalog_log_to_graphite.sh

### PR DESCRIPTION
**Pull request by JasonIamAUnixAdmin**
Added new script to send info to graphite server

### CODE REVIEW

File: monitor/etc/sensu/plugins/send_catalog_log_to_graphite.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+#gdate used on OS X 
+#date=/opt/local/bin/gdate
+date=/bin/date
+
+nc=/usr/bin/nc
+
+
+#LOG=/Users/jschuster/peanut_wrk/logs/wrk
+LOG=/mnt/data/catalog_wrapper.log
```
Comment (cpenner461) [inline monitor/etc/sensu/plugins/send_catalog_log_to_graphite.sh]: Can we make this a variable required at runtime?

File: monitor/etc/sensu/plugins/send_catalog_log_to_graphite.sh [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+#LOG_TAIL=/Users/jschuster/.virtualenvs/peanut_wrk/bin/pygtail
+LOG_TAIL=/bin/pygtail
+METRIC_PATH=sensu.peanut_cloud_htspotlight_com.catalog
```
Comment (cpenner461) [inline monitor/etc/sensu/plugins/send_catalog_log_to_graphite.sh]: Can we make these variables required at runtime? (METRIC_PATH and GRAPH_HOST)

**Comment by cpenner461** [comment]
In addition to the couple of specific comments, can we name the script something like "log_metrics.sh" and clean up a few of the extraneous commented lines?

**Comment by JasonIamAUnixAdmin** [comment]
I've made the requested changes. Added flags and help for them

**Pull request merged**